lib/dap/filter/simple.rb

```diff
@@ -171,6 +171,20 @@ def process(doc)
   end
 end
 
+class FilterFlatten
+  include Base
+  def process(doc)
+    self.opts.each_pair do |k,v|
```

Comment:
Minor, but you aren't using `v` in this block so maybe switch to `self.opts.each_key do |k|`